src/class-permissions-helper.php

```diff
@@ -192,4 +192,19 @@ public function post_type_has_admin_bar( $post_type ) {
 
 		return $post_type_object->public && $post_type_object->show_in_admin_bar;
 	}
+
+	/**
+	 * Determines whether a Rewrite & Republish copy can be republished.
+	 *
+	 * @param \WP_Post $post The post object.
+	 *
+	 * @return bool Whether the Rewrite & Republish copy can be republished.
+	 */
+	public function is_copy_allowed_to_be_republished( \WP_Post $post ) {
+		if ( $post->post_status === 'dp-rewrite-republish' || $post->post_status === 'private' ) {
+			return true;
+		}
+
+		return false;
```

Comment:
Is it intentional to not just have (for readability maybe?)
`return $post->post_status === 'dp-rewrite-republish' || $post->post_status === 'private'` ?

Reply:
No objections. I was also considering to change it in:
`\in_array( $post->post_status, [ 'dp-rewrite-republish', 'private' ], true )`

Reply:
Also fine with me!